web/packages/teleport/src/services/api/api.ts

```diff
@@ -99,9 +99,9 @@ const api = {
       headers: Record<string, string>;
       mfaResponse: MfaChallengeResponse;
       signal: AbortSignal;
-    }> &
-      (// Either data or formData.
-      | { data?: unknown; formData?: never }
+    }> & // Either data or formData.
+      (
+        | { data?: unknown; formData?: never }
```

Comment:
Something weird is happening with Prettier here, I ran it twice and it resulted in two different outcomes as you can see by the latest commits.